pkg/nfd-worker/nfd-worker.go

```diff
@@ -169,7 +165,7 @@ func newDefaultConfig() *NFDConfig {
 	return &NFDConfig{
 		Core: coreConfig{
 			LabelWhiteList: utils.RegexpVal{Regexp: *regexp.MustCompile("")},
-			SleepInterval:  duration{60 * time.Second},
+			SleepInterval:  utils.DurationVal{Duration: 60 * time.Second},
```

Comment:
You could leave out the field name (as a sidenote the Regexp above is also unnecessary)

```suggestion
			SleepInterval:  utils.DurationVal{60 * time.Second},
```

Reply:
It made my linter angry xd saying that `DurationVal struct literal uses unkeyed fields`

Reply:
K, I was wrong then. Let's leave it as it is. Might be the reason that the regexp is also like that 😅